Mudit_Gupta/progress.md

```diff
@@ -3,78 +3,80 @@
 **Project: Usability Hero**
 
 Overall Direction:
-- 
-- 
-- 
+- Provide Context specific help
+- Improve Getting Started guide
+- Improve documentation structure
+- Sidebar for sections of instructor help
+- Resolve UI/UX issues
 
 ---
 
 Week 1 Plan:
 - [ ] 
-- [ ] 
-- [ ] 
+- [ ] Work on [Student: feedback submission: distribution points question: explain if equal points are not allowed](https://github.com/TEAMMATES/teammates/issues/8817)
+- [ ] Discuss with Cara further work to be done on Getting started guide and help docs
```

Comment:
I was keeping the first week light because I had an interview on 21st which was scheduled before GSoC results came out so was trying to prepare for it simultaneously.

Yes writing tests along with the component makes sense I will change that and distribute the workload as well.

Reply:
uhh i see. Wokay I guess for the first week u can just do the simple issue and discuss with cara...all the best for ur interview 👍 please update the plan once u have discussed with cara...today is alr Wednesday...